### DISCUSSION TITLE
Hi all. What does this text mean: Your explorer listing, PROJECT NAME, has been reviewed, but has been rejected.

### DISCUSSION THREAD

**Discussion by SobeitB** [WalletConnect/walletconnect-monorepo#2352] [Developer Support]
Your explorer listing, PROJECT NAME, has been reviewed, but has been rejected.

Reason: Thank you for your application, after careful review and testing, we were unable to complete the following tests. Please troubleshoot, retest and submit your changes so that we can review and approve the request for you: Indicated v1 support for WalletConnect Sign SDK - Test integration using our test dapp for wallets (https://example.walletconnect.org/) Indicated v2 support for WalletConnect Sign SDK - Test integration using our test wallet for dapps (https://react-web3wallet.vercel.app/) Indicated v1 support for WalletConnect Auth SDK - Test integration using our test wallet for dapps (https://react-web3wallet.vercel.app/)

Please refer to our Explorer Guidelines for more information. https://explorer.walletconnect.com/guidelines

Best regards,
WalletConnect Cloud Team

**Comment by finessevanes** [comment]
Hey - sorry for the generic email blast there, we're updating our process in how we share feedback. Can you give me your project name and project ID?

**Comment by finessevanes** [comment]
Closing this discussion due to three weeks of inactivity.
